tfgrid3/umbrel/dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:stable-slim
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    UMBREL_VERSION="release" \
+    UMBREL_REPO="getumbrel/umbrel" \
+    UMBREL_INSTALL_PATH="/umbrel" 
+
+RUN apt update && \
```

Comment:
Do not use apt as it is meant to be an end-user tool (interactive tool), use apt-get instead which is more suitable for non-interactive usage in scripts. this applies for all apt commands, e.g. `apt update`, `apt install`, etc